homeassistant/components/upnp/sensor.py

```diff
@@ -99,6 +100,11 @@ class UpnpSensorEntityDescription(UpnpEntityDescription, SensorEntityDescription
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
+    UpnpSensorEntityDescription(
+        key=PORT_MAPPING_NUMBER_OF_ENTRIES_IPV4,
+        translation_key="port_mapping_number_of_entries_ipv4",
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
```

Comment:
Should this be added as default?
```suggestion
    UpnpSensorEntityDescription(
        key=PORT_MAPPING_NUMBER_OF_ENTRIES_IPV4,
        translation_key="port_mapping_number_of_entries_ipv4",
        entity_category=EntityCategory.DIAGNOSTIC,
        entity_registry_enabled_default=False,
    ),
```